tests/pages/package-edit.js

```diff
@@ -32,8 +39,12 @@ import Datepicker from './datepicker';
 }
 
 @interactor class PackageEditDropDownMenu {
+  addToHoldings = new Interactor('.tether-element [data-test-eholdings-package-add-to-holdings-action]');
+  removeFromHoldings = new Interactor('.tether-element [data-test-eholdings-package-remove-from-holdings-action]');
   clickRemoveFromHoldings = clickable('.tether-element [data-test-eholdings-package-remove-from-holdings-action]');
```

Comment:
We should consider removing these, maybe making a note that they're deprecated? because instead of

```js
interactor.clickRemoveFromHoldings();
```

we can say
```js
interactor.removeFromHoldings.click()
```